sdk/nodejs/billingCycle.ts

```diff
@@ -12,8 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export const HourlyBilling:  BillingCycle = "hourly";
-export const MonthlyBilling: BillingCycle = "monthly";
```

Comment:
If we made a release with these in we should probably keep them for a while and mark as `@deprecated`?

Reply:
@jen20 none of these constants are in the public yet - just in master